### DISCUSSION TITLE
NTFS Support for Cosmos

### DISCUSSION THREAD

**Discussion by Doofus-Mc-Goofus** [CosmosOS/Cosmos#3025] [Ideas]
I don't know if this is already a thing in cosmos, but I would like NTFS support.

**Comment by Gabolate** [comment]
Cosmos dosen't have NTFS support yet, there is this project [here](https://github.com/Twometer/LineOS) that implements NTFS but its a bit old so i don't know if it still works

**Reply by zarlo** [reply]
it should still work

**Reply by cabfile** [reply]
(it actually doesnt)
